modules/ref-cpu-metrics.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: REFERENCE
+
+[id='ref-cpu-metrics_{context}']
+= CPU metrics
+
+[role='_abstract']
+You can track metrics on your CPU to ensure the health of the hardware.
+
+CPU utilization::
+--
+Tracks CPU usage to identify workloads that are CPU-bound.
+
+_Query_
+[source,bash]
+----
+sum(rate(container_cpu_usage_seconds_total{namespace="granite", pod=~"granite.*"}[5m])) by (namespace)
```

Comment:
Please make sure to clarify that the namespace and the pod are configurable in the beginning of the dashboard in the variables section.
I'll update the `kustomize` file I am creating for this.

The queries will looks like something similar to:
```
vllm:gpu_cache_usage_perc{namespace="$namespace", pod=~"granite.*", model_name="$model_name"}
```

The podname will be added as part of the kustomize command to apply the dashboard.